cppForSwig/Makefile

```diff
@@ -40,23 +42,33 @@ else
 PYTHON_INCLUDES=$(shell $(PYVER)-config --includes )
 endif
 
-CPPFLAGS += $(ARMORY_CPPFLAGS) -Icryptopp -Imdb -DUSE_CRYPTOPP -D__STDC_LIMIT_MACROS
-LDLIBS += -lpthread -Lmdb
-SWIG_OPTS    += -c++ -python -threads
-
-SWIG_INC     += 
-
-CXXCPP += $(CPPFLAGS) $(PYTHON_INCLUDES) -std=c++11
+# "-Wl,--no-as-needed" used for Ubuntu 12.04 due to rt library linking issues.
+# http://stackoverflow.com/questions/17150075/undefined-reference-to-clock-gettime-although-lrt-is-given
+# has more info. Require only for static linking. When 12.04 goes, remove this
+# unless it's discovered that OS X needs it.
+ifdef STATIC_LINK
+LDLIBS += -Wl,--no-as-needed
```

Comment:
@josephbisch - Finally taking a look at it. (Sorry. Felt a bit ill for most of the day.) I've been having talks about this and hope to make a final decision tomorrow. Some at Armory feel that compiler flags are fine. Maybe I'm just not doing a good enough job of explaining the situation? Oh well. I think this code is going to get rearranged eventually anyway, so your call would be touched only by 64-bit builds.